cassandra/cluster.py

```
@@ -1605,34 +1595,33 @@ def _create_thread_pool_executor(self, **kwargs):
         tpe_class = ThreadPoolExecutor
-        if sys.version_info[0] >= 3 and sys.version_info[1] >= 7:
-            try:
-                from cassandra.io.eventletreactor import EventletConnection
-                is_eventlet = issubclass(self.connection_class, EventletConnection)
-            except:
-                # Eventlet is not available or can't be detected
-                return tpe_class(**kwargs)
+        try:
+            from cassandra.io.eventletreactor import EventletConnection
+            is_eventlet = issubclass(self.connection_class, EventletConnection)
+        except:
```

Ruff (0.16.0)

[error] Do not use bare `except`

(E722)